### PR TITLE
correct document spelling issues

### PR DESCRIPTION
**Pull request by yanmxa**
Signed-off-by: myan [myan@redhat.com](mailto:myan@redhat.com)

### CODE REVIEW

File: content/zh/getting-started/integration/cluster-proxy.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ feature called [konnectivity egress-selector](https://kubernetes.io/docs/tasks/e
 which is majorly for setting up a TCP-level proxy for kube-apiserver
 to get access to the node/cluster network. Here are a few terms we need
 to clarify before we elaborate on how the cluster proxy resolve multi-cluster
-control plan network connectivity for us:
+control plane network connectivity for us:
```
Comment (dhaiducek) [inline content/zh/getting-started/integration/cluster-proxy.md:45]: Please also update the `content/en` directory with this change.

File: content/en/concepts/managedcluster.md [hunk 2]
```diff
@@ -56,7 +56,7 @@ command-line tool `clusteradm` to manage the whole registration process.
 When we're registering a new cluster into OCM, the registration agent will be
 starting by creating an unaccepted `ManagedCluster` into the hub cluster along
 with a temporary [CertificateSigningRequest (CSR)](https://kubernetes.io/docs/reference/access-authn-authz/certificate-signing-requests/)
-resource. The cluster will be accepted by the hub control plan, if the
+resource. The cluster will be accepted by the hub control plane, if the
```
Comment (dhaiducek) [inline content/en/concepts/managedcluster.md:59]: Please also update the `content/zh` directory with this change.

**Review by dhaiducek — CHANGES_REQUESTED** [review]
Thanks for the PR, @yanmxa! Would you please make the equivalent changes in the `en` and `zh` folders?

File: content/zh/getting-started/integration/cluster-proxy.md (hunk 1 above)
Comment (yanmxa) [reply content/zh/getting-started/integration/cluster-proxy.md:45]: Done!

File: content/en/concepts/managedcluster.md (hunk 2 above)
Comment (yanmxa) [reply content/en/concepts/managedcluster.md:59]: It's right in the `content/zh`.

**Review by mikeshng — APPROVED** [review]
/approve

/lgtm

**Pull request merged**